handlers/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: drush clear cache
```

Comment:
I already made it `drush cache clear drush`, the second suggestion from your first comment.  Can we stick with that?  If you care, I can change it again.

Maybe there should be a handler to clear the site cache.  How would we make sure that it applied to the correct site?  This feels like issue creep ...